Libplanet/Action/ValidatorStateExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Libplanet.Consensus;
+
+namespace Libplanet.Action
+{
+    internal static class ValidatorStateExtensions
+    {
+        public static ValidatorSet GetValidatorSet(this IAccountStateDelta delta)
```

Comment:
AFIK, placing `public` members onto `internal` class can be a little bit debated ([see also](https://ericlippert.com/2014/09/15/internal-or-public/) 

but [just for consistency](https://github.com/planetarium/libplanet/blob/0.46.0/Libplanet/Action/AccountStateDeltaExtensions.cs#L12), I support using `internal` atm. 😕

Reply:
Many libplanet classes now using `internal` extension but `public` methods. (`NetMQFrameExtensions`, `BytesConvertExtensions`, `StunAddressExtensions`, `ActionEvaluationsExtensions`, `EnumerableExtensions`, `KeyValuePairExtensions` and many more..)

Reply:
I see... then, it'd be good to incorporate them in another PR or discussion.